sys/include/fido2/ctap/ctap_mem.h

```diff
@@ -41,28 +41,15 @@ extern "C" {
 /** @} */
 
 /**
- * @brief First flash page to store data in
- *
- * @note This can corrupt firmware if CTAP_FLASH_START_PAGE is set to a
- * flash page containing firmware. Therefore make sure that CTAP_FLASH_START_PAGE
- * is located after the firmware.
+ * @brief   Default amount of flashpages to use
  */
-#if defined(CONFIG_FIDO2_CTAP_FLASH_START_PAGE) && \
-    (CONFIG_FIDO2_CTAP_FLASH_START_PAGE >= 0)
-#define CTAP_FLASH_START_PAGE CONFIG_FIDO2_CTAP_FLASH_START_PAGE
-#else
-#define CTAP_FLASH_START_PAGE (FLASHPAGE_NUMOF - 4)
+#ifndef CONFIG_FIDO2_CTAP_NUM_FLASHPAGES
+#define CONFIG_FIDO2_CTAP_NUM_FLASHPAGES 4
 #endif
 
-/**
- * @brief Start page for storing resident keys
- */
-#define CTAP_FLASH_RK_START_PAGE CTAP_FLASH_START_PAGE
-
-/**
- * @brief Page for storing authenticator state information
- */
-#define CTAP_FLASH_STATE_PAGE CTAP_FLASH_RK_START_PAGE - 1
+#if CONFIG_FIDO2_CTAP_NUM_FLASHPAGES <= 2
```

Comment:
Why smaller **equal**? One for state + at least one for rks? It also contradicts the Kconfig range in this way.

Reply:
Correct that is wrong. Ty!